packages/react-components/react-text/src/components/Caption1/Caption1.tsx

```diff
@@ -21,10 +21,10 @@ const useStyles = makeStyles({
 });
 
 /**
- * Text wrapper component for the Caption typography variant
+ * Text wrapper component for the Caption1 typography variant
  */
-export const Caption: FunctionComponent<TextWrapperProps> = createWrapper({
+export const Caption1: FunctionComponent<TextWrapperProps> = createWrapper({
```

Comment:
Besides the regular `Caption1`, typography spec also contains `Strong` and `Stronger` variants.
![image](https://user-images.githubusercontent.com/9615899/168115307-d4033122-2d07-4290-b140-12894732f808.png)
The same applies to some other Types as well. Do you plan to add those as well either through a separate components or a prop?

Reply:
Just check Text design spec as well, these Type presets are there as well.

Reply:
huh, I see, thanks!